packages/smooth_app/lib/pages/onboarding/consent_analytics_page.dart

```diff
@@ -16,84 +14,48 @@ class ConsentAnalytics extends StatelessWidget {
   Widget build(BuildContext context) {
     final Size size = MediaQuery.of(context).size;
     final AppLocalizations appLocalizations = AppLocalizations.of(context)!;
-    const String assetName = 'assets/onboarding/analytics.svg';
```

Comment:
Could you remove this file from the assets, please?

Reply:
✅